src/flags.ts

```diff
@@ -7,7 +7,7 @@
 
 export const DEFAULT_FLAGS: ReadonlyArray<string> = [
   // Disable built-in Google Translate service
-  '--disable-translate',
+  '--disable-features=TranslateUI',
```

Comment:
is this going to be a problem for folks that are passing their own `--disable-features` if it's supposed to be a csv? should we be combining them?

Reply:
> should we be combining them?

probably yes. doesn't seem a high priority but yah